src/gen_kotlin/templates/module.kt

```diff
@@ -62,12 +62,13 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
     {% endif -%}
     {%- endfor %}  
     @ReactMethod
-    fun setLogStream(promise: Promise) {
+    fun setLogStream(filterLevel: String, promise: Promise) {
```

Comment:
I think this should be optional
```suggestion
    fun setLogStream(filterLevel: String?, promise: Promise) {

```